addon/components/froala-editor.js

```diff
@@ -24,14 +24,24 @@ export default Ember.Component.extend({
         this.set('_froala', froala);
     },
 
-    handleFroalaEvent: function(event, editor) {
-      var events = this.get('eventNames');
+    handleFroalaEvent: function(event, editor,x,y,z) {
       const eventName = event.namespace;
       const reverseEventName = eventName.replace(/\./g,"_");
+      const actionHandler = this.attrs[reverseEventName||"image_beforeUpload"];
```

Comment:
`image_beforeUpload` appears as being hardcoded. Isn't there a better way doing it?